code/modules/clothing/suits/miscellaneous.dm

```diff
@@ -617,6 +617,54 @@
 	armor = list("melee" = 10, "bullet" = 0, "laser" = 0, "energy" = 0, "bomb" = 0, "bio" = 0, "rad" = 0, "fire" = 0, "acid" = 0)
 	hoodtype = /obj/item/clothing/head/hooded/winterhood/miner
 
+/obj/item/clothing/head/hooded/ablative
+	name = "ablative hood"
+	desc = "Hood hopefully belonging to an ablative trenchcoat. Includes a visor for cool-o-vision."
+	icon_state = "ablativehood"
+	armor = list("melee" = 10, "bullet" = 10, "laser" = 60, "energy" = 50, "bomb" = 0, "bio" = 0, "rad" = 0, "fire" = 100, "acid" = 100)
+	strip_delay = 30
+	var/hit_reflect_chance = 50
+
+/obj/item/clothing/head/hooded/ablative/equipped(mob/living/carbon/human/user, slot)
+	..()
+	to_chat(M, "As you put on the hood, a visor shifts into place and starts analyzing the people around you. Neat!")
+	ADD_TRAIT(user, TRAIT_SECURITY_HUD, CLOTHING_TRAIT)
+
+/obj/item/clothing/head/hooded/ablative/dropped(mob/living/carbon/human/user)
+	..()
+	to_chat(M, "You take off the hood, removing the visor in the proces and disabling its integrated hud.")
+	REMOVE_TRAIT(user, TRAIT_SECURITY_HUD, CLOTHING_TRAIT)
+
+/obj/item/clothing/head/hooded/ablative/IsReflect(def_zone)
+	if(!(def_zone in BODY_ZONE_HEAD)) //If not shot where ablative is covering you, you don't get the reflection bonus!
+		return FALSE
+	if (prob(hit_reflect_chance))
+		return TRUE
+
+/obj/item/clothing/suit/hooded/ablative
+	name = "ablative trenchcoat"
+	desc = "Experimental trenchcoat specially crafted to reflect and absorb laser and disabler shots. Don't expect it to do all that much against an ax or a shotgun, however."
+	icon_state = "ablativecoat"
+	item_state = "ablativecoat"
+	body_parts_covered = CHEST|GROIN|LEGS|FEET|ARMS|HANDS
+	cold_protection = CHEST|GROIN|LEGS|FEET|ARMS|HANDS
+	heat_protection = CHEST|GROIN|LEGS|FEET|ARMS|HANDS
+	armor = list("melee" = 10, "bullet" = 10, "laser" = 60, "energy" = 50, "bomb" = 0, "bio" = 0, "rad" = 0, "fire" = 100, "acid" = 100)
+	hoodtype = /obj/item/clothing/head/hooded/ablative
+	strip_delay = 30
+	equip_delay_other = 40
+	var/hit_reflect_chance = 50
+
+/obj/item/clothing/suit/hooded/ablative/Initialize()
+	. = ..()
+	allowed = GLOB.security_vest_allowed
+
+/obj/item/clothing/suit/hooded/ablative/IsReflect(def_zone)
+	if(!(def_zone in list(BODY_ZONE_CHEST, BODY_ZONE_PRECISE_GROIN, BODY_ZONE_L_ARM, BODY_ZONE_R_ARM, BODY_ZONE_L_LEG, BODY_ZONE_R_LEG))) //If not shot where ablative is covering you, you don't get the reflection bonus!

```

Comment:
why not if(def_zone in list(BODY_ZONE_HEAD etc

Reply:
because that would need updating should new zones ever get added.

Reply:
because the hood is a separate item? what

Reply:
oh